Migrations/migrations.sql

```
@@ -0,0 +1,243 @@
+        [id] [int] IDENTITY(1,1) NOT NULL,
+        [name] [nvarchar](100) NOT NULL,
+        [role] [varchar](15) NULL,
+        [contact_id] [int] NOT NULL,
+        PRIMARY KEY CLUSTERED ([id] ASC)
+    )
+END
+GO
+
+-- Add FK and CHECK constraints to users
+IF NOT EXISTS (
+    SELECT * FROM sys.foreign_keys WHERE name = 'FK_restaurant_contact'
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)